scripts/configure_dev.sh

```diff
@@ -41,6 +41,45 @@ function install_or_upgrade {
     fi
 }
 
+function install_windoows_shellcheck() {
```

Comment:
This function is not needed here, since a file is added for the same purpose: scripts/windows/install_shellcheck.sh

Reply:
Done